src/base/utils/StandardModel.jl

```diff
@@ -76,7 +77,8 @@ Return a dictionary mapping the flux of atoms through a reaction in `model`.
 function atom_exchange(rxn_id::String, model::StandardModel)
     atom_flux = Dict{String,Float64}()
     for (met, stoich_rxn) in model.reactions[rxn_id].metabolites
-        adict = get_atoms(model.metabolites[met])
+        adict = metabolite_formula(model, met)
+        isnothing(adict) && continue
```

Comment:
I so much wish there was a function for this like with haskell, this is double `traverse`, and you don't need to care about whether it's nothing or empty or whatever. :D